nautobot/docs/configuration/authentication/ldap.md

```diff
@@ -84,8 +84,21 @@ AUTH_LDAP_BIND_PASSWORD = "demo"
 LDAP_IGNORE_CERT_ERRORS = True
 ```
 
+### TLS Options
+
 STARTTLS can be configured by setting `AUTH_LDAP_START_TLS = True` and using the `ldap://` URI scheme.
 
+Apply TLS settings to the internal SSL context on nautobot by configuring `ldap.OPT_X_TLS_NEWCTX` with value `0`. 
+
+```
```

Comment:
It's not clear to me here... Is `AUTH_LDAP_START_TLS = True` also required for this to work? If so, would you mind including it in the example config you provided, along w/ the `ldap://` URI for the service?

Reply:
Yes will include the commands. TLS can be configured as described or using `ldaps://` URI and `AUTH_LDAP_START_TLS = False`, which is a little less intuitive.